addons/stock_picking_batch/models/stock_move_line.py

```diff
@@ -88,3 +90,246 @@ def _add_to_wave(self, wave=False):
         if picking_to_wave_vals_list:
             self.env['stock.picking'].create(picking_to_wave_vals_list)
         wave.action_confirm()
+
+    def _is_auto_wave_batchable(self):
+        self.ensure_one()
+        return not (not self.picking_id
+           or self.state != 'assigned'
+           or not self.quantity
```

Comment:
should use `float_is_zero`with `precision_rounding=self.product_uom_id.rounding` to check if 0 quantity